config/locales/proper.en.yml

```diff
@@ -81,6 +88,8 @@ en:
     # PROTOCOL #
     ############
     protocol:
+      new_research_study: "New Research Study"
```

Comment:
This value is pulled from `t(:protocols)[:studies][:new]` in the application

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/service_requests/protocol/_protocol_display.html.haml